index.js

```diff
@@ -17,10 +17,16 @@ function parseOptions(options) {
 }
 
 // Get `<a>` element as string
-const linkify = (match, options) => {
-	const fullReference = match.replace(/^#/, `${options.user}/${options.repository}#`);
-	const [userRepository, issue] = fullReference.split('#');
-	const href = `${options.baseUrl}/${userRepository}/issues/${issue}`;
+const linkify = (reference, options) => {
+	const {
+		organization = options.user,
+		// Optional repository isn't actually supported by the regex:
+		// https://github.com/sindresorhus/issue-regex/issues/17
+		repository = options.repository,
+		issueNumber,
+	} = issueRegex().exec(reference).groups;
```

Comment:
Part 1: use `issueRegex` to extract data from the string instead of recreating it via split/replace